<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewScene</class>
 <widget class="QWidget" name="ViewScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QSpinBox" name="SetViewchoice">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="RebuildCubbe">
       <property name="text">
        <string>重建异形体</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="ResetRot">
       <property name="text">
        <string>重置旋转角</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="4" colspan="2">
      <widget class="QGraphicsView" name="graphicview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="BuildFromView">
       <property name="text">
        <string>从三视图建立异形体</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="DisplayExpand">
       <property name="text">
        <string>显示展开图</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="text">
        <string>返回菜单</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="DisplayThreeviews">
       <property name="text">
        <string>切换为生成模式</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="KeepRatioWidget" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="savecube">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="readcube">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeepRatioWidget</class>
   <extends>QFrame</extends>
   <header location="global">keepratiowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
